### PR TITLE
deploy: ovm-pair cli options

### PR DESCRIPTION
**Pull request by tynes**
Adds CLI options for specifying the L1 and L2 providers

### CODE REVIEW

File: publish/src/commands/deploy-ovm-pair.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const L1_PROVIDER_URL = 'http://localhost:9545';
 const L2_PROVIDER_URL = 'http://localhost:8545';
```
Comment (K-Ho) [inline publish/src/commands/deploy-ovm-pair.js:14]: Should remove these lines if they're no longer used

File: publish/src/commands/deploy-ovm-pair.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ const L1_PROVIDER_URL = 'http://localhost:9545';
...
 const DATA_PROVIDER_URL = 'http://localhost:8080';
 
-const deployOvmPair = async () => {
+const deployOvmPair = async ({ l1ProviderUrl, l2ProviderUrl }) => {
```
Comment (K-Ho) [inline publish/src/commands/deploy-ovm-pair.js:17]: add dataProviderUrl here!

**Review by K-Ho — CHANGES_REQUESTED** [review]
couple tiny changes

**Review by K-Ho — APPROVED** [review]
lgtm, let's wait for an approval from a Synthetix team member!

**Comment by tynes** [comment]
@K-Ho I think its probably fine to merge this into your PR so that a Synthetix team member only has to do 1 code review on a single PR

**Pull request merged**